### PR TITLE
Add new tile for Generative AI readings to precourse

### PR DESCRIPTION
**Pull request by kelsey-steven-ada**
Adds a new tile to the precourse to inform students that there will be an in class discussion, share questions for reflection that will be used in the in-class activity, and link to articles that give high level information about LLMs and Generative AI.

This PR addresses this ticket: https://app.asana.com/0/1201700438643002/1205762659804145/f
The asana ticket also covers creating [a shareable doc for the in-class activity](https://docs.google.com/document/d/1F6MAomC65s9oAzCHg3yh57yrYHoCIo88jOoeCfI6qHk/edit).

### CODE REVIEW

File: generative-ai-overview/generative-ai-prework.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    * What kinds of impacts can that have when the responses are used to make decisions?
+3. What kinds of tasks are LLMs good at performing? What kinds of tasks do they struggle with? 
+4. What are a few considerations for using AI tools responsibly?
+
+## Required Readings
+What are LLMs?
+* [Introduction to Large Language Models](https://developers.google.com/machine-learning/resources/intro-llms) - From Google's Machine Learning resources
+
+How does bias impact LLMs?
+* *Content Warning: there are no graphic depictions, but there are examples discussed of harmful sexist and racist content created by generative AIs* [Exploration of bias in ChatGPT and other AI models](https://www.insider.com/chatgpt-is-like-many-other-ai-models-rife-with-bias-2023-1) - from Insider.com
```
Comment (apradoada) [inline generative-ai-overview/generative-ai-prework.md:22]: I like the addition of the content warning here.

File: generative-ai-overview/generative-ai-prework.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+We will be introducing topics around Generative AI and showcasing some tools and techniques for working with them throughout the coursework, so we want to create a shared understanding of generally what an LLM is and how it operates. We acknowledge that students may have different levels of experience and interest with these topics before starting at Ada. Some folks may find it helpful to skim through the materials while others may want to do a couple close readings. 
+
+## Questions for Reflection
+
+However you choose to move through the materials, as you read through these resources, try to reflect on the following questions for our in-person discussion:
+1. Given a text prompt, how do LLMs like chatGPT try to create a response? 
+    * How does this differ from how people create responses when asked a question?
+2. How does the data a LLM was trained on affect the responses that they create? 
+    * What kinds of impacts can that have when the responses are used to make decisions?
+3. What kinds of tasks are LLMs good at performing? What kinds of tasks do they struggle with?
```
Comment (apradoada) [inline generative-ai-overview/generative-ai-prework.md:14]: I love all these questions. One subquestion I could see here is something along the lines of "What are the technical limitations of LLMS?" In terms of discussing the cost (financial and time-wise) of LLMs. Definitely worded better than what I have but I think that could be a good jumping off point to get students thinking about the limits of technology.

**Review by apradoada — COMMENTED** [review]
I think this looks great, Kelsey! I just had one potential addition. Feel free to add it in or not!

File: generative-ai-overview/generative-ai-prework.md (hunk 2 above)
Comment (kelsey-steven-ada) [reply generative-ai-overview/generative-ai-prework.md:14]: I've added the following, let me know if this encapsulates what you were thinking =]

> 5. What are some technical or physical limitations of LLMs?
>     * What are they able to process? What equipment or infrastructure does it take to keep an LLM running?

**Pull request closed without merging**